src/app/workspace/resource/resource-instance-form/resource-instance-form.component.ts

```diff
@@ -372,4 +378,34 @@ export class ResourceInstanceFormComponent implements OnInit, OnDestroy {
 
     }
 
+    /**
+     *
+     * Determines whether the selected resource class has a binary Representation (image etc.).
+     *
+     * Returns the file value type or an empty array.
+     *
+     * @param properties the properties to check for binary file value types.
+     */
+    private _hasBinaryRepresentation(properties: ResourcePropertyDefinition[]): string[] {
+
+        const binaryTypes = [Constants.StillImageFileValue, Constants.AudioFileValue, Constants.DocumentFileValue, Constants.MovingImageFileValue];
+
+        // remove file value properties from property list
+        this.properties = properties.filter(
+            prop => !binaryTypes.includes(prop.objectType)
+        );
+
+        // TODO: add a FormControl representing the file value to propertiesParentForm
```

Comment:
@flavens @mpro7 I added [Outputs to the upload file component](https://github.com/dasch-swiss/dsp-ui-lib/pull/264).

My idea is that on a successful file upload, a `CreateStillImageFileValue` can be created. However, I am not sure where this should happen. Would `SelectPropertiesComponent` be the right place to handle this?

I have added some logic for the upload to `ResourceInstanceFormComponent`, but maybe this should be moved to  `SelectPropertiesComponent`.

Reply:
Or maybe instead of the Outputs,  `UploadFileComponent`'s form could be used: as long as there is no successful upload to Sipi, the form is invalid.  `UploadFileComponent` would have to be added to `propertiesParentForm` then.

But then instead of the binary file (`File`), the `UploadedFileResponse` obtained from Sipi should be stored in the `FormControl`.

Reply:
@tobiasschweizer IMO FormControl has to be used anyway, so second options sounds good for me.

Reply:
ok, I'll try that approach next week.